### PR TITLE
Add cnames for remaining 18F guides

### PR DESCRIPTION
**Pull request by echappen**
**IS BLOCKED BY #672** 

- [ ] This is a new public-facing site _(if so, please follow the additional instructions below)_
   - [ ] Provide context
   - [ ] Assign to `@18F/osc` for review
   - [ ] [TTS Digital Council's new site review process](https://docs.google.com/document/d/1j6eieL3oop0rxCAldVVh7uGdOCG-ajafrog_BZ-u470/edit) completed
   - [ ] Update [the inventory](https://docs.google.com/spreadsheets/d/1OBO6g7_OsVBv0vG8WSCI6L2FD_iRh3A7a_6eQWj2zLE/edit?ts=6025575d#gid=2013137748) with new site information

### CODE REVIEW

File: terraform/18f.gov.tf [hunk 1]
```diff
@@ -1545,36 +1512,30 @@ resource "aws_route53_record" "d_18f_gov__acme-challenge_federalist-proxysite-te
...
-  name    = "ux-guide.18f.gov."
-  type    = "A"
-
-  alias {
-    name                   = "d2mhwjcivqpysk.cloudfront.net."
-    zone_id                = local.cloud_gov_cloudfront_zone_id
-    evaluate_target_health = false
-  }
+  name    = "_acme-challenge.ux-guide.18f.gov."
+  type    = "TXT"
```
Comment (apburnes) [inline terraform/18f.gov.tf:1518]: Is there a reason this is a TXT and not a CNAME?

Comment (echappen) [reply terraform/18f.gov.tf:1518]: There were a few TXT records already present in the file (there's one for the engineering guide as well). Should I be removing the TXT records for the guides?

Comment (apburnes) [reply terraform/18f.gov.tf:1518]: There should be no TXT records for `_acme-challenge.ux-guide.18f.gov` subdomain.

**Review by apburnes — APPROVED** [review]
👍

**Review by apburnes — APPROVED** [review]

**Pull request merged**